bmc-acf/acf_manager.cpp

```diff
@@ -88,14 +88,13 @@ acf_info ACFCertMgr::installACF(std::vector<uint8_t> accessControlFile)
     }
     // Verify and install ACF and get expiration date.
     Tacf tacf{[](std::string msg) {
-        log<phosphor::logging::level::INFO>(msg.c_str());
+	    lg2::info(msg.c_str());
```

Comment:
lg2 can take strings, so you don't need the c_str().

Reply:
Thanks, I've removed the unnecessary conversion.  The code looked like this (lambda function):
> [](std::string msg) { lg2::info(msg); }

In doing so, a funny thing happened.   The C++ compiler complained that I was (1) declaring msg again, shadowing the parameter, (2) the parens around (msg) were unnecessary, and (3) I was using deleted function 'lg2::info<>::info()'.
So I added a literal format string.

Reply:
I think the lg2 author didn't want people passing in the whole message as an arg.